apps/codeforafrica/src/components/Navigation/index.js

```diff
@@ -0,0 +1,39 @@
+import { NavBar, Section } from "@commons-ui/core";
+import { useTheme } from "@mui/material/styles";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import PropTypes from "prop-types";
+import React from "react";
+
+import DesktopNavigation from "@/codeforafrica/components/Navigation/DesktopNavigation";
+import MobileNavigation from "@/codeforafrica/components/Navigation/MobileNavigation";
+
+function Navigation({ menu }) {
```

Comment:
@kilemensi  Not sure about this ...we already have a NavBar component imported from common-ui...might result in something like this 👇🏽 
<img width="477" alt="Screenshot 2022-05-13 at 10 53 49" src="https://user-images.githubusercontent.com/3693563/168237407-156e902d-32f3-4973-a451-f8f599f54c92.png">

Reply:
@kilemensi Something like this would work `import { NavBar as NavigationBar, Section } from "@commons-ui/core"`. then name rename the component `Navigation` to `NavBar`

Reply:
Exactamundo @nerdlet